mesheryctl/internal/cli/root/system/context/context.go

```diff
@@ -26,8 +26,8 @@ import (
 	"github.com/spf13/viper"
 )
 
-var configuration config.MesheryCtlConfig
-
+var configuration *config.MesheryCtlConfig
+var tempContext string = "local"
```

Comment:
In addition to the various context configurations that are being attempted, if we haven't already, it would be good to incorporate use of the same context template that we use when `system context create` is executed.

Reply:
@leecalcote  Inside of tests, I don't see the use of that template context when I am using hardcoded golden files. Where exactly did you ask to incorporate that template context inside of tests?